packages/config/config/devices/0x019b/z-dim2.json

```diff
@@ -0,0 +1,255 @@
+{
+	"manufacturer": "Heatit",
+	"manufacturerId": "0x019b",
+	"label": "Z-DIM2",
+	"description": "Heatit Z-Dim2",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x2201",
+			"zwaveAllianceId": 4666
+		},
+		{
+			"productType": "0x0002",
+			"productId": "0x2201"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Switch Multilevel",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Power Restore Level",
+			"description": "Range between 0(off) and 99(max)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Return to last level",
+					"value": 100
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "Load Dimming Mode",
+			"description": "Choose the dimming type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Trailing edge",
+					"value": 0
+				},
+				{
+					"label": "Leading edge",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Maximum Dim Level",
+			"description": "Max dim level, range 2-99",
```

Comment:
```suggestion
```
Unnecessary description, please refer to the style guide:
https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=parameter-descriptions
> As a rule of thumb: Only include a description if it is necessary, helpful, and adds significant value.